alicloud/resource_alicloud_cs_kubernetes.go

```diff
@@ -1051,6 +1064,13 @@ func buildKubernetesArgs(d *schema.ResourceData, meta interface{}) (*cs.Delicate
 		}
 	}
 
+	var apiAudiences string
+	if list := expandStringList(d.Get("api_audiences").([]interface{})); len(list) > 0 {
+		apiAudiences = strings.Join(list, ",")
+	} else {
+		apiAudiences = ""
+	}
```

Comment:
The `apiAudiences` defaults to "" and the line 1070 is needless.

Reply:
Done!